### PR TITLE
add-cart-ordering

### PR DESCRIPTION
**Pull request by paulchrisluke**
paid_via should probably link to payment method types
prefixing "order_" or orders is silly
products being JSON is kind of ick HANDLE THE ICK
order should probably link to OrderLine which is made up of products
Products -> Cart -> Copy to Order + Line Items on checkout

### CODE REVIEW

File: api/orders/models/orders.settings.json [hunk 1]
```diff
@@ -18,25 +18,19 @@
...
     },
-    "order_phone_number": {
-      "type": "string"
-    },
-    "order_paid_via": {
+    "phone_number": {
       "type": "string"
     },
-    "order_tracking": {
+    "tracking": {
```
Comment (StrangeWill) [inline api/orders/models/orders.settings.json:33]: What is "tracking" though?

**Review by StrangeWill — APPROVED** [review]
There are a lot of questions on things like "how do migrations work in the future", which could influence decisions on this (eg: the payment method table), but I think this may be okay for now.

File: api/orders/models/orders.settings.json (hunk 1 above)
Comment (paulchrisluke) [reply api/orders/models/orders.settings.json:33]: that was from their tutorial. I assume if it had a tracking/shipping ID?

Comment (StrangeWill) [reply api/orders/models/orders.settings.json:33]: Hah, possibly... but that depends on how you do shipping, sometimes you do that on the line item.

**Pull request merged**